qa/tasks/cephadm.py

```diff
@@ -12,7 +12,9 @@
 import re
 import uuid
 
+import six
 from ceph_manager import CephManager
+import toml
```

Comment:
Can the imports be alphabetical order?

Reply:
imports have to be in a specific order:

1. imports from standard libraries
2. imports from other libraries
3. imports form your project

Reply:
For example, toml should go after `from teuthology` and `import six` right before `from tarfile`

Reply:
> imports have to be in a specific order:
> 
> 1. imports from standard libraries
> 2. imports from other libraries
> 3. imports form your project

So why do you mix `six`, `toml` and `ceph_manager` and `teuthology`?

Reply:
fixed